### PR TITLE
XW-3550 | css alignment adjustments for Page Header

### PR DESCRIPTION
**Pull request by weronika**
https://wikia-inc.atlassian.net/browse/XW-3550

### CODE REVIEW

**Review by latata — APPROVED** [review]

File: extensions/wikia/PageHeader/styles/index.scss [hunk 1]
```diff
@@ -49,14 +48,15 @@
 	}
 
 	&__languages {
-		padding-bottom: $text-dropdown-padding-bottom;
+		margin-bottom: 12px;
```
Comment (rybmat) [inline extensions/wikia/PageHeader/styles/index.scss:51]: @weronika why did you change it from 10px back to 12px? it is not consistent with categories dropdown
